src/utils/graph/solver.js

```diff
@@ -75,15 +75,25 @@ const solveStrict = (constraints, constants) => {
     addVariable(co.b, co.base.property);
   }
 
+  let unsolvableCount = 0;
+
   for (const co of constraints) {
-    solver.addConstraint(
-      co.base.strict(
-        co,
-        constants,
-        variables[variableId(co.a, co.base.property)],
-        variables[variableId(co.b, co.base.property)]
-      )
-    );
+    try {
+      solver.addConstraint(
+        co.base.strict(
+          co,
+          constants,
+          variables[variableId(co.a, co.base.property)],
+          variables[variableId(co.b, co.base.property)]
+        )
+      );
+    } catch (err) {
+      unsolvableCount += 1;
+    }
+  }
+
+  if (unsolvableCount > 0) {
+    console.warn(`Skipped ${unsolvableCount} unsolvable constraints`);
```

Comment:
Is there no more information worth spitting out to the console?

Reply:
Yea, would be nice to improve this error message some how

Reply:
So I did experiment with this, but essentially the problem is that the constraint systems involved are not very human interpretable given their size and complexity, such that the individual constraints here themselves don't really directly relate to the cause of the user's problem in a meaningful way.

I think for now the notification will be enough to keep an eye on given we're not expecting to see it after this fix, but if there are further complications still then we can invest more into somehow making this more interpretable.

Reply:
Could we at least put the notification on screen rather than in the console?